### ISSUE TITLE
Place holder - DRS/SPL PDF form for no Web Access Complainant

### ISSUE THREAD

**Issue by randyabramson** [usdoj-crt/crt-portal-management#1004]
More info coming after meeting with Killian on 8/5/21

- Randy has notes

**Comment by randyabramson** [comment]
@blaughman  @emilycryan:

Some notes from 8/4/21 meeting:
Emily's EOS samples:
* https://www.justice.gov/opa/press-release/file/1417796/download
* https://www.justice.gov/opa/press-release/file/1417801/download

Next steps from 8/4/21 meeting

- [ ] Bill to check on who we can contact at OMB to make sure that if we use the Portal as a basis, we won't need approval later.
- [ ] Emily to evaluate branching questions and assess how many forms we need
- [ ] Evaluate if we need translations
- [ ] Emily continue to work with IT (Terri or someone else) to make PDFs accessible

**Comment by blaughman** [comment]
FCS used/uses this complaint form

https://www.justice.gov/file/1259441/download

**Comment by emilycryan** [comment]
@blaughman do any other sections have their own form? And I'm assuming if we took this one down for FCS and used one mirroring the portal, they'd be OK with that...? Just want to make sure we aren't going backwards in terms of getting back to unique forms for sections.

CC @randyabramson

**Comment by blaughman** [comment]
@emilycryan I'll blast an email to the sections.  I sent an email to Linda Quash about the FCS complaint form - I'm hoping she can chat today.  Also, I'm not sure how FCS will react to a standardized complaint form.  When Randy and I met with their Chief, Christine Stoneman, she indicated she wanted a pathway for complaints to come directly to FCS.  At this point, their is no pathway with the current routing logic.  
CC @randyabramson

**Comment by emilycryan** [comment]
Ahhh I worked with Christine for the Language Access / Equity project. The upside is that we would be able to provide translated PDFs which I know is important to her based on our research findings. So maybe that's our carrot :D 

Thanks @blaughman and @randyabramson

**Comment by emilycryan** [comment]
@blaughman Would those filling this out mail to the same address as on the portal? 


U.S. Department of Justice
Civil Rights Division
950 Pennsylvania Avenue, NW
Washington, D.C. 20530-0001

And what email should they submit an electronic version to? CC @randyabramson

**Comment by blaughman** [comment]
@emilycryan That is the correct address.  They could email it to Ask.CRT@usdoj.gov.  One issue with emailing to that address is the auto reply which tells them to file their complaint in the portal.  
CC @randyabramson

**Comment by emilycryan** [comment]
Ok, first pass at this. I would like to be cognizant of how much extra info we put in this form packet so the last page includes the instructions and privacy policy info since the initial use of this is as an email attachment. LMK thoughts and if you think addl info should be included. General design is aligned to what we're doing elsewhere and what's been approved so it can't really change - I also want to stay away from additional forms and see if this branched approach might work. CC @randyabramson @blaughman 

[CRT-PDF Form Rework-Aug 2021.pdf](https://github.com/usdoj-crt/crt-portal-management/files/6963716/CRT-PDF.Form.Rework-Aug.2021.pdf)

**Comment by blaughman** [comment]
@emilycryan Amazing work, as always.  Only suggestion I have would be to include, if possible, the helper text that is currently on the web for the location name.  The intake staff indicated the public's response to this required information is sometimes lacking.
"Examples: Name of business, school, intersection, prison, polling place, website, etc."
CC @randyabramson

**Comment by emilycryan** [comment]
@blaughman Good catch - also my program put in periods in a couple of unnecessary spots. Here's the updated version. CC @randyabramson 

[CRT-PDF Form Rework-Aug 2021.pdf](https://github.com/usdoj-crt/crt-portal-management/files/6963943/CRT-PDF.Form.Rework-Aug.2021.pdf)

**Comment by randyabramson** [comment]
Looks great, @emilycryan! Question: do we want to include the new text around 'supporting materials?'

**Comment by emilycryan** [comment]
@randyabramson do you think the text is relevant here? Seems weird on a paper form since we don't need to "limit" them and in fact, we encourage them to use additional pages so telling them they cannot attach supporting docs feels contradictory - if they can print off something of value to their report and include it, why not let them?We don't have the technical limitations that exist on the reporting portal and Glynis has said in our interviews that they do that already. CC @blaughman

**Comment by randyabramson** [comment]
@emilycryan right...it doesn't make sense to mention attachments on the form. I was thinking more the part about 'if you provide details, we will know what will be most helpful...'

Not necessary if you don't think it's helpful.

**Comment by emilycryan** [comment]
@randyabramson Lemme think about it and see if a rewording is better for this one. In the meantime, can you and @blaughman review the instructions / final page? I think that is really important but I want to be really mindful of the real estate on it. One other thought is to make that content a sort of "intro" page and then use this final page to include the info we've been discussing as a final/next steps page but again, wanted your inputs. Killian said we had to have the privacy policy with links to the website's longer form so unfortunately, that needs to be present, even if in a small font. But we had explored other content as well and before I lay it out, is there anything you guys think we *must* have either as intro or conclusion here? I'd say a max of one page intro and max one page final would be the absolute limit (and even that is pushing it a bit with two pages of pure content out of a total 10....)

**Comment by blaughman** [comment]
@emilycryan I have no issue with the final page.  The instructions are clear and accurate.  @randyabramson Should we run this by Dan for awareness?

**Comment by randyabramson** [comment]
@blaughman  - yes, good call.

@emilycryan - I think this looks great! 

Question: Should link to the instructions on page 1? Top line could read: "If you believe you or someone else has experienced a civil rights violation, please tell us what happened. Before starting, you can also read <ul>instructions<ul> for this form and what to expect after submission.' or the like????

**Comment by emilycryan** [comment]
Great suggestions @randyabramson I've added in the text and when we are close to ready to go, I can add in the hyperlink to the page (you'll see it underlined in the doc but the hyperlink isn't set up just yet) I also added page numbers which can help in case anyone calls in and has issues, as well as allow the person filling it out to put the pages in order. CC @blaughman 
[CRT-PDF Form Rework-Aug 2021.pdf](https://github.com/usdoj-crt/crt-portal-management/files/6975909/CRT-PDF.Form.Rework-Aug.2021.pdf)

**Comment by randyabramson** [comment]
Looks great, @emilycryan! I know Dan is OOO until next week, so it might be a bit before we can get this in front of him. cc: @blaughman

**Comment by emilycryan** [comment]
Yes, agree @randyabramson - the good news is that he's generally onboard with the designed approach and has been pushing me / others to do more of this across the board for DOJ materials so I would imagine we will get his approval. Since we have a week or so, is there anything we should consider jump starting in the meantime? Do we want to get this in front of sections or review in deep dive...? I hate to lose a week if we can do something in the meantime. CC @blaughman

**Comment by randyabramson** [comment]
@emilycryan if you have time and wanna keep rolling on this, I am cool with that.

A couple of things: 
- We mentioned above that we wanted to make this DRS PDF accessible with Isabelle. Should we get that rolling?
- We mentioned that we also wanted to work on a form with branching questions. Maybe SPL would be a good one to tackle next, since so many of the prisoners don't have Web access?

**Comment by emilycryan** [comment]
@randyabramson Isabelle generally prefers to do this once so I'd like to wait until it's final/approved. And this likely won't take her but a couple of hours to do what she needs to do. 

As for branching... this design *is* branched. That's why the primary complaint is structured the way it is with the branched questions included in the specific primary complaint boxes and the "location only" design in step 3. We do not need specific branched forms with this design. Plus we said we preferred a singular form b/c otherwise we're going back to the days of multiple forms which negates the experience we've worked so hard to achieve. 

This form is intended to work for everyone - I remember Kilian saying we must maintain generality in the form if we want to avoid having to get a new PRA. The goal of this form is to be printed or filled out online and saved for attaching to an email so it's already usable for those w/o web access. 

Let me know if you disagree or have more questions around this! Happy to talk it through. 
CC @blaughman

**Comment by randyabramson** [comment]
Next step is to let Terri review. Then send to Isabelle, right @emilycryan ?

**Comment by emilycryan** [comment]
@randyabramson We still need the PRA info to include in the bottom right corner but yes, this should be correct in terms of next steps. 

CC @blaughman

**Comment by randyabramson** [comment]
Waiting on PRA from Melody

**Comment by emilycryan** [comment]
@randyabramson Melody's response isn't required for this - the PRA is the same from the website which Dan sent last week and has been updated on the PDF that was sent to Terri. So we're good on this. CC @blaughman

**Comment by randyabramson** [comment]
@emilycryan I believe Dan sent the PRA number for this. Next step is to send this to Isabelle, right?

**Comment by emilycryan** [comment]
@randyabramson Yes, as I mentioned above, Dan sent it, it's been updated on the form and we have sent it to Terri. And we chatted last week and came to the conclusion that Isabelle is not the person for this work, it's Terri and she has the form. CC @blaughman

**Comment by randyabramson** [comment]
@emilycryan Sorry...I forgot to follow up with you on this today.

I am attaching Terri's final here. Anything else need to be done?

I not, I asssume the next steps are to get them linked on the ADA and beta.ada.gov site (cc @blaughman)

[CRT-ReportPDF-Sep 2021-508.pdf](https://github.com/usdoj-crt/crt-portal-management/files/7205532/CRT-ReportPDF-Sep.2021-508.pdf)

**Comment by emilycryan** [comment]
@randyabramson Yes I think we are good to link it up! I'm still working on the Tagalog form but we should be able to make it work which means not having to change anything on this english version. Let's get it launched if you and @blaughman are ready to do that.

**Comment by randyabramson** [comment]
Also, @emilycryan I see the OMB number on the doc has "x's" and the expiration date is not 04/30/24 (as it is in the doc that Dan sent)

**Comment by emilycryan** [comment]
@randyabramson I'm seeing the correct info on the one you posted and the one Terri sent. Here it is again: 
[CRT-ReportPDF-Sep 2021-508.pdf](https://github.com/usdoj-crt/crt-portal-management/files/7205776/CRT-ReportPDF-Sep.2021-508.pdf)
